tools/send_email_to_user.py

```
@@ -0,0 +1,115 @@
+
+    # 使用设置中的“收件人列表”别名发送
+    python tools/send_email_to_user.py --to 小明 --subject "测试" --body "你好"
+
+    # 从文件读取正文并附带附件
+    python tools/send_email_to_user.py --to user@example.com --subject "报告" \
+        --body-file report.txt --attachment a.png --attachment b.png
+
+    # 使用 html/test_email.html 模板发送测试邮件
+    python tools/send_email_to_user.py --test-email
+
+    # 不使用 SSL（例如使用 587 STARTTLS）
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)